src/types/index.d.ts

```diff
@@ -0,0 +1,8 @@
+declare global {
+  interface Window {
+    wts: {
+      push(props: [string, string, { [key: string]: string }]): void;
```

Comment:
Please improving the documentation and specification for this.

Examples of better specification:
- The initial string should always be "send"
- The second string should be either "page", "link", "click" or "pageupdate"
- The third string is not simply an object with string keys and values. Values may be objects in itself.

Linking to sources like the following would be helpful:

- https://documentation.mapp.com/1.0/en/manual-track-request-25105181.html
- https://documentation.mapp.com/1.0/en/how-to-send-manual-tracking-requests-page-updates-7240681.html

Reply:
Very good thinking. Thanks! 🧠 💡